app/src/main/java/com/github/sdpteam15/polyevents/model/Zone.kt

```diff
@@ -1,8 +1,10 @@
 package com.github.sdpteam15.polyevents.model
 
+import android.util.Log
 import com.github.sdpteam15.polyevents.database.DatabaseConstant.ZoneConstant.*
 import com.google.android.gms.maps.model.LatLng
 import com.google.firebase.firestore.IgnoreExtraProperties
+import java.lang.Exception
 
 /**
  * Entity model for a zone. Events occur inside a zone.

```

Comment:
I just noticed that the zone model has its value location as String that can be parsed into squares.
Is that a good choice? I think we should store them as List<List<LatLng>> directly and then parse into a string in "ZoneAdapter".

Moreover, I don't think storing Zones as serialized string is a good choice again, you can store data in lists on firebase.
This would be much more clean.

Reply:
I created an issue, we'll fix this in an other sprint